src/mesh/features/rbac.md

```diff
@@ -71,6 +72,57 @@ rules:
 ```
 {% endnavtab %}
 {% endnavtabs %}
+{% endnavtab %}
+{% navtab `targetRef` selectors %}
+For policies using `targetRef` selector. You can specify which `targetRef` kinds users should have access to.
+
+{% navtabs %}
+{% navtab Kubernetes %}
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: AccessRole
+metadata:
+  name: role-1
+spec:
+  rules:
+  - types: ["MeshTrafficPermission", "MeshTrace", "MeshAccessLog"] # list of types to which access is granted. If empty, then access is granted to all types
+    names: ["res-1"] # list of allowed names of types to which access is granted. If empty, then access is granted to resources regardless of the name.
+    mesh: default # Mesh within which the access to resources is granted. It can only be used with the Mesh-scoped resources.
+    access: ["CREATE", "UPDATE", "DELETE"] # an action that is bound to a type.
+    when: # a set of qualifiers to receive an access. Only one of them needs to be fulfilled to receive an access
+    - tagetRef: # a condition on targetRef section in policies 2.0 (like MeshAccessLog or MeshTrace).
+        kind: MeshService
+        name: backend
+    - targetRef:
+        kind: MeshSubset
+        tags: 
+        - name: k8s.kuma.io/namespace
+          value: kuma-demo
+```
+{% endnavtab %}
+{% navtab Universal %}
+```yaml
+type: AccessRole
+name: role-1
+rules:
+- types: ["MeshTrafficPermission", "MeshTrace", "MeshAccessLog"] # list of types to which access is granted. If empty, then access is granted to all types
```

Comment:
I think all the suggestions I had in the last tab above would apply to this tab as well.

Reply:
Updated second tab